contracts/boost-claim/src/contract.rs

```diff
@@ -0,0 +1,86 @@
+#![no_std]
+#![allow(unused_attributes)]
+
+use address_boost_info::AddressBoostInfo;
+use multiversx_sc::imports::*;
+use multiversx_sc_modules::only_admin;
+
+pub mod address_boost_info;
+pub mod config;
+
+#[multiversx_sc::contract]
+pub trait BoostClaimContract: config::ConfigModule + only_admin::OnlyAdminModule {
+    #[init]
+    fn init(
+        &self,
+        difference_between_claims: u64,
+        levels_prizes: MultiValueEncoded<ManagedBuffer>,
+    ) {
+        self.set_difference_between_claims(difference_between_claims);
+        self.set_levels_prizes(levels_prizes);
+    }
+
+    #[upgrade]
+    fn upgrade(&self) {}
+
+    #[only_admin]
+    #[endpoint(setDifferenceBetweenClaims)]
+    fn set_difference_between_claims(&self, difference_between_claims: u64) {
+        self.time_difference_in_seconds()
+            .set(difference_between_claims);
+    }
+
+    #[only_admin]
+    #[endpoint(setLevelsPrizes)]
+    fn set_levels_prizes(&self, levels_prizes: MultiValueEncoded<ManagedBuffer>) {
```

Comment:
Add to config file as this is a setup endpoint.
Also, I'd suggest 1 of 3 approaches here. In order to avoid getting into data inconsistency due to a different number of levels, you can either:

1. Only add to the end of the current vector.
2. Replace all the levels as you do right now, but add a check that the new array is at least the same length to the current one. (this approach is better in case you may need to change the saved text for some levels).
3. Delete this endpoint and only allow the levels setup during init. In case you may need a change, you'll upgrade the contract and prepare the custom logic once you have the desired behavior defined.